data/org.freedesktop.Flatpak.xml

```diff
@@ -152,6 +152,21 @@
       <arg type='s' name='installation' direction='in'/>
     </method>
 
+    <method name="CancelPull">
+      <arg type='u' name='flags' direction='in'/>
+      <arg type='s' name='installation' direction='in'/>
+      <arg type='s' name='src_dir' direction='in'/>
+    </method>
+
+    <method name="GetRevokefsFd">
+     <annotation name="org.gtk.GDBus.C.UnixFD" value="true"/>
```

Comment:
Out of interest, does the generated code really not have the `GUnixFDList` parameters without this? That seems pretty sad since an argument of type `h` should be enough to tell the code generator it needs them… but if that's not already the case, I guess we can't fix it without breaking existing code.

Reply:
Yes, the `annotation` is required. The generator won't use the `GUnixFdList` dbus-call method otherwise.

Reply:
I filed https://gitlab.gnome.org/GNOME/glib/issues/1726 about this. But I did notice that the `<annotation>` here is misaligned relative to the other child elements of `<method`.